locales/en/messages.json

```diff
@@ -1120,7 +1120,7 @@
         "message": "Configure via the BlackBox tab after enabling."
     },
     "featureRX_SPI": {
-        "message": "SPI RX support"
+        "message": "SPI (e.g. built-in Rx)"
```

Comment:
The correct name here is "SPI Rx", this is how named in most of tutorials, and users knows with this name.

Reply:
Indeed, however the "Rx" part seems unnecessary, given that this is the connection mode list for the Rx.  
SPI is indeed used for other things like gyro, but here we know we are referring to SPI Rx only, it can be no other thing.
None the less I will add Rx back, it just seems unnecessary waste of screen.
It would be like adding Rx after CRSF, or after PWM.
They are all Rx modes.